src/drivers/cmakeFileApiDriver.ts

```diff
@@ -269,6 +269,10 @@ export class CMakeFileApiDriver extends CMakeDriver {
             log.showChannel();
             log.info(proc.buildCmdStr(this.cmake.path, args));
             return 0;
+        } else if (this.isMultiConfig && trigger === ConfigureTrigger.setVariant) {
```

Comment:
Does the code in the else (line 275) still need to run if the only change is to prevent a reconfigure?